MatrixSDK/Data/Store/MXFileStore/MXFileStore.m

```diff
@@ -382,6 +383,16 @@ - (void)deleteRoom:(NSString *)roomId
 - (void)deleteAllData
 {
     MXLogDebug(@"[MXFileStore] Delete all data");
+    
+    if (self.clearSecondaryStoresCallback)
+    {
+        self.clearSecondaryStoresCallback();
+        [MXStoreHelper secondaryStoresWereClearedFor:credentials.userId];
+    }
+    else
+    {
+        [MXStoreHelper setSecondaryStoresToBeClearedFor:credentials.userId];
+    }
```

Comment:
Hmmm just realised that we seem to have multiple instances of the file store at times. So whilst the data needs to be deleted, I'm not sure if this will be helpful with the timing 😕